Orion.Net.Client/Scripts/ScriptParameterInterpreter.cs

```diff
@@ -32,7 +32,7 @@ public static List<ScriptParameterInterpreterResult> ExtractParams(this string p
                 else
                 {
                     // Error to be managed :
-
+                    
                 }
```

Comment:
Is it really necessary ?
At worst, a new ScriptParameterInterpreterResult is not add and the classes calling this fonction already check if there is a parameter or send a message